dimagi/pages/templates/sections/blog/full_post.html

```diff
@@ -13,7 +13,7 @@ <h1 class="gap-bottom-base">
     <p>
       {% include 'partials/social.html' with url=request.build_absolute_uri text=post.title %}
     </p>
-    <div class="post-content-text pad-bottom-small">
+    <div class="post-content-text pad-bottom-small epic-content">
```

Comment:
what's the reason for adding the `epic-content` tag? this was originally reserved for pillar pages.

Reply:
As in the ticket they mentioned they need spacing as the following link (https://dimagi.com/blog/mobile-data-collection-how-to-choose-the-right-tool/)  the above link page using the class epic-content.